### ISSUE TITLE
Clean reinstall in Windows?

### ISSUE THREAD

**Issue by bryanjuho** [netbirdio/netbird#1731]
Hi, I'm trying to clean reinstall my netbird Windows client.
I've tried deleting with the uninstaller + C:\Program Data\Netbird\config.json , but after installing again, my settings still shows values from the previous installation.

Are there registries or other locations where configurations / cache persists even after uninstall?

Thanks

**Comment by arobinsongit** [comment]
@bryanjuho what was your solution to this?  I am having the same issue.

**Comment by bryanjuho** [comment]
> @bryanjuho what was your solution to this?  I am having the same issue.

I'm sorry but I was working on a Windows VM so I just clean installed it again.
Sorry i couldnt help.
Best of luck.

**Comment by arobinsongit** [comment]
@bryanjuho the solution I just came up with was to delete the peer that was connected to the original source in Netbird and then run another install/setup.  It works but not great.  It's obvious something is getting cached somewhere but can't quite figure out what.

**Comment by Marcus1Pierce** [comment]
I found another workaround that gave me a fully clean Netbird installation on Windows. Here’s what I did:

1. Uninstall Netbird. Remove the Netbird application using the standard Windows uninstaller.
2. Delete the Netbird data folder. Go to C:\ProgramData\ and delete the Netbird folder.
3. Reinstall Netbird. Download and install the latest version of Netbird as if it were a fresh setup.
4. Re-authenticate your device. Sign in again to register the device as a new peer.
